python/cudf_polars/cudf_polars/experimental/benchmarks/pdsh.py

```diff
@@ -25,13 +25,22 @@
 from typing import TYPE_CHECKING, Any
 
 import numpy as np
-import pynvml
 
 import polars as pl
 
-from cudf_polars.dsl.translate import Translator
-from cudf_polars.experimental.explain import explain_query
-from cudf_polars.experimental.parallel import evaluate_streaming
+try:
+    import pynvml
+except ImportError:
+    pynvml = None
+
+try:
+    from cudf_polars.dsl.translate import Translator
+    from cudf_polars.experimental.explain import explain_query
+    from cudf_polars.experimental.parallel import evaluate_streaming
+
+    CUDF_POLARS_AVAILABLE = True
```

Comment:
You can guard any uses of cudf_polars in this module with `if CUDF_POLARS_AVAILABLE`